### ISSUE TITLE
Removing branding from admin themes

### ISSUE THREAD

**Issue by whitepaperclip** [smpallen99/ex_admin#272]
Any thoughts on allowing users to change footer/header text content through config options?

**Comment by gwincr11** [comment]
This should help

Add a custom logo:
```
config :ex_admin,
  logo_full: "<h1>Logo</h1>",
  logo_mini: "<h1>Logo</h1>"
```


Add custom header info like a stylesheet:
```
config :ex_admin,
  head_template: {ExAdminDemo.AdminView, "admin_layout.html"}

```
  Where:
  * `ExAdminDemo.AdminView` is a view in your project
  * `admin_layout.html` is a template in `web/templates/admin` directory

This renders in the head of the page, so you can put in stylesheets

**Comment by whitepaperclip** [comment]
I did this for the header. I ended up using some JS snippet to remove the footer:

```javascript
$("#footer").html("Powered by ExAdmin");
```

Ideally we wouldn't have to do this, though.

**Comment by whitepaperclip** [comment]
Addressed in pull request #309
